### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by KovalevArtem** [OpenCovidTrace/octrace-ios#4]
It will be great if during the release the application will support different languages!
I am ready to help in the translation into Russian.
(For example, you can use Weblate, or any other service. It is completely free for open source projects.)

**Comment by alexnivanov** [comment]
@KovalevArtem thanks for the interest, we'll address this issue ASAP!

**Comment by KovalevArtem** [comment]
Then I'm sure you will need @orangesunny of Weblate.
He can help set everything up.

**Comment by alexnivanov** [comment]
I guess, as the site says that it ain't free :D

**Comment by KovalevArtem** [comment]
> I guess, as the site says that it ain't free :D

Gratis for libre projects

**Comment by alexnivanov** [comment]
Yeah, I found free sign-up now via Pricing tab - was not that obvious. Will do after finishing implementing some critical features...

**Comment by nathamsr11** [comment]
It will be great if during the release the application will support different languages!
I am ready to help in the translation into french.

**Comment by alexnivanov** [comment]
Submitted hosting request to Weblate, waiting for their answer...
